test/integration/main_test.go

```diff
@@ -39,6 +40,33 @@ func KubeClient() (result *kubernetes.Clientset) {
 	return kubeClient
 }
 
+func COInsights(k *kubernetes.Clientset) map[string]interface{} {
+	// get info about insights cluster operator
+	data, err := k.RESTClient().Get().AbsPath("/apis/config.openshift.io/v1/clusteroperators/insights").DoRaw()
+	obj := map[string]interface{}{}
+	err = json.Unmarshal(data, &obj)
+	if err != nil {
+		panic(err.Error())
+	}
+	return obj
+}
+
+func IsOperatorDegraded(t *testing.T, config map[string]interface{}) bool {
```

Comment:
Probably a nitpick, but it would be IMHO better to have a docstring (comment) for all exported function. In this case, it is possible we don't have to export these functions outside current test package?

Reply:
well, I see I was wrong about exported functions and it's not needed here as everything is in one package (at least now).